docs/source/set_coords.rst

```diff
@@ -0,0 +1,22 @@
+Set Coords
+===========
+
+.. image:: _static/images/set_coords.png
+
+This panel provides the option to set and reset the coordinates.
```

Comment:
Refer to these as tabs. Which reminds me, why is Panel and hvPlot not mentioned anywhere in this documentation??